gym_wrapper/base_gym_env.py

```diff
@@ -199,6 +275,8 @@ def __build_human_render_image(self):
             return img
 
         image_list = [game_state.screen_buffer]
+        if self.channels == 1:
+            image_list = [np.repeat(game_state.screen_buffer[..., None], repeats=3, axis=2)]
```

Comment:
Why repetition? If we only have a single-channel image, we should probably just return a single-channel image instead of replicating it over multiple dimensions.

Reply:
Yeah I guess it doesn't need to, I was just following what was already done with the depth buffer below this, but I'm assuming that had to be done because the rgb and automap were `[H, W, 3]` and thus the depth buffer was made to match?

Reply:
Hmm depth buffer seems to be without a channel (i.e. a single channel), no stacking involved. For consistency and clarity, you could add the channel dimension to all observations: `depth` buffer would become `[H, W, 1]` instead of just `[H, W]`. `image` and `automap` would also always have channel dimension, but it would vary between 1 and 3 depending on which screen format was chosen.

Reply:
Alright in my local version of this pr I did this (making the default observation `[H, W, 1]` for `depth`, `labels` buffer. Definitely agree this is better for consistency.

But for the rendering component (`__build_human_render_image()`) you do need to repeat the color channels so that there are always 3 `[H, W, 3]` for each buffer for 2 reasons:
1. The current way the renderer is implemented using pygame always expects rgb images `[H, W, 3]` and breaks with a single color channel `[H, W, 1]` 
```
surf = pygame.surfarray.make_surface(render_image)
  File "/home/bebeal/anaconda3/envs/doomDev/lib/python3.8/site-packages/pygame/surfarray.py", line 100, in make_surface
    return pix_make_surface(array)
ValueError: must be a valid 2d or 3d array
```
2. If the labels buffer is active then it's an outlier in that is always an rgb image `[H, W, 3]` regardless of screen type and since we stack the buffers horizontally to render the rest of them have to have the same shape `[H, W, 3]`. 

The repeat hack does work for all combinations of buffers:
![render](https://user-images.githubusercontent.com/42706447/175252548-a59cae74-6470-4208-9c40-53a90e1b6194.png)

Reply:
So I mean we could change it to avoid repeating, but it's a little more involved. I think repeating for now is fine unless you think otherwise.